qiskit/visualization/state_visualization.py

```diff
@@ -511,109 +510,105 @@ def plot_state_city(
         ax1 = None
         ax2 = ax_imag
 
-    max_dzr = max(dzr)
-    min_dzr = min(dzr)
-    min_dzi = np.min(dzi)
+    fig.tight_layout()
+
+    max_dzr = np.max(dzr)
     max_dzi = np.max(dzi)
 
-    # There seems to be a rounding error in which some zero bars are negative
-    dzr = np.clip(dzr, 0, None)
+    # Figure scaling variables since fig.tight_layout won't work
+    fig_width, fig_height = fig.get_size_inches()
+    max_plot_size = min(fig_width / 2.25, fig_height)
+    max_font_size = int(3 * max_plot_size)
+    max_zoom = 10 / (10 + np.sqrt(max_plot_size))
 
-    if ax1 is not None:
-        fc1 = generate_facecolors(xpos, ypos, zpos, dx, dy, dzr, color[0])
-        for idx, cur_zpos in enumerate(zpos):
-            if dzr[idx] > 0:
-                zorder = 2
-            else:
-                zorder = 0
-            b1 = ax1.bar3d(
-                xpos[idx],
-                ypos[idx],
-                cur_zpos,
-                dx[idx],
-                dy[idx],
-                dzr[idx],
+    print(real_color, imag_color)
```

Comment:
this looks like debugging print

Reply:
Yes it was. I have made a commit [7ca633](https://github.com/Qiskit/qiskit/pull/10590/commits/87ca63330f27f3a3e731fc94aa61a0c56f71cf78) to remove it. Thank you!